syne_tune/backend/backend.py

```diff
@@ -196,7 +196,7 @@ def stop_all(self):
 
     def set_path(self, results_root: Optional[str] = None, tuner_name: Optional[str] = None):
         """
-        :param results_root: the parent that should contains the results of all experiments
+        :param results_root: the parent that should contains the results of the tuning experiment.
```

Comment:
Could you add a comment on what tuner_name is doing? For example, for the SM backend, it determines how SM jobs are called.

Reply:
I also do not understand one thing: Is the SageMakerBackend using results_root when set_path is called? If not, why not?

Reply:
> I also do not understand one thing: Is the SageMakerBackend using results_root when set_path is called? If not, why not?

No it does not since the SM backend does not store local files (as opposed to local backend). 



> Could you add a comment on what tuner_name is doing? For example, for the SM backend, it determines how SM jobs are called.

There is a comment "we use the tuner-name to set the checkpoint directory" in `SageMaker Backend`, I also added another comment in `Backend` on `set_path`.

Can you approve again?